packages/react-data-grid-addons/src/formatters/ImageFormatter.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+interface Props {
+  /** image url, used as background-image */
+  value: string;
+}
+
+export default function ImageFormatter({ value }: Props) {
+  return <div className="react-grid-image" style={{ backgroundImage: `url(${value})` }} />;
```

Comment:
The previous `ImageFormatter` implementation loaded the url in the background before applying the style, I don't know why, doesn't seem worthwhile.